mlir/test/Dialect/OpenMP/ops.mlir

```diff
@@ -133,6 +133,85 @@ func.func @omp_parallel_pretty(%data_var : memref<i32>, %if_cond : i1, %num_thre
   return
 }
 
+// CHECK-LABEL: omp_loop_nest
+func.func @omp_loop_nest(%lb : index, %ub : index, %step : index) -> () {
+  // TODO Remove induction variables from omp.wsloop.
+  omp.wsloop for (%iv) : index = (%lb) to (%ub) step (%step) {
```

Comment:
[remark] I fear there will be inconcistencies if potentially both `omp.loop_nest` and `omp.wsloop` both define potentially inconsistent loop bounds and intermediate passes not considering the wrapper passes. Since we already discussed this, only a remark here.